src/Illuminate/Auth/Console/stubs/make/views/layouts/app.stub

```diff
@@ -15,9 +15,9 @@
 
     <!-- Scripts -->
     <script>
-        window.Laravel = {!! json_encode([
-            'csrfToken' => csrf_token(),
-        ]) !!};
+        window.Laravel = {
+            csrfToken: document.querySelector('meta[name="csrf-token"]').getAttribute('content')
```

Comment:
Why would you give more responsibility to the client when it can be easily used as it was?